doc/rofi-theme.5.markdown

```diff
@@ -1010,8 +1010,8 @@ By default the `element-icon` and `element-text` child widgets are added to the
 `element`. This can be modified using the `children` property or the
 `[no]-show-icons` option.
 
-A child added with another name is seen as a `box`, this can be used as dynamic
-padding to tweak how an entry looks when it expands to available space.
+A child added with another name is treated the same as the special widget described
+in the [advanced layout](https://github.com/davatorium/rofi/blob/next/doc/rofi-theme.5.markdown#advanced-layout) section.
```

Comment:
Maybe not an absolute link, but relative in current document?

> [advanced layout](#advanced-layout)

Reply:
Done. I didn't know if the manpage generation could handle relative links ;).